language/compiler/bytecode_source_map/src/disassembler.rs

```diff
@@ -76,6 +85,145 @@ impl<Location: Clone + Eq + Default> Disassembler<Location> {
             .struct_def_at(struct_definition_index))
     }
 
+    //***************************************************************************
+    // Formatting Helpers
+    //***************************************************************************
+
+    fn name_for_field(&self, field_idx: FieldDefinitionIndex) -> Result<(String, String)> {
```

Comment:
Would it make sense to have a separate `name_for_field` and `type_for_field`?

Reply:
I think it makes sense if we break up the `name_for_local` to do the same here.